pages/projects/osg-software.md

```diff
@@ -24,25 +24,58 @@ software systems that comprise the OSG fabric of production services.
     [DOMA group](../focus-area/doma.md) to lead the transition to a bearer token-based authentication and authorization
     infrastructure and HTTP/S-based transfer protocol.
 
+    -   **Retirement of OSG 3.5:** OSG 3.5 was the last version of the OSG Software Stack with Globus dependencies
+        and its retirement marks the completion of the years-long transition from X.509 to token-based pilot job
+        authentication.
+        In particular, this meant the end of OSG Software support for GridFTP, GSI, Hadoop, and LCMAPS.
+        This version of the software stack provided the bridge between the two different authentication methods,
+        supporting both concurrently, and allowed for sites to become more familiar with the new technologies.
+
+    -   **Release of OSG 3.6** OSG 3.6 is the first version of the OSG Software Stack without any Globus
+        dependencies.
+        Triggered by the 2017 announcement that Globus was ending support for their open-source Globus Toolkit, the
+        release of OSG 3.6 is the culmination of years of planning and development, and includes versions of HTCondor,
+        HTCondor-CE, and XRootD that rely on token-based authentication instead of X.509 proxy authentication.
+        In addition to the changes in software, the team made a number of release policy changes to ease the transition
+        to token-authentication and to reflect the increasingly agile operational models being explored by the LHC.
+
 -   **Container images:** the OSG Software and Release team has supported the LHC community’s exploration into DevOps
     deployment models by distributing select production services as container images.
-    In particular, the team developed and maintains the [caching](caching.md) container image used by multiple LHC experiments.
-    The team follows a strategy of providing a common base container across all communities, plus community-specific refinements.
-    The team currently supports three communities, U.S. ATLAS, U.S. CMS, and the general OSG community, with [caching](caching.md) container images.
+    In particular, the team developed and maintains the [caching](caching.md) container image used by multiple LHC
+    experiments.
+    The team follows a strategy of providing a common base container across all communities, plus community-specific
+    refinements for U.S. ATLAS and U.S. CMS.
+
+-   **Enterprise Linux 8 support:** the OSG Software Stack is built upon the Enterprise Linux (EL) operating system
+    substrate and new operating system versions requires rebuilding and verification of the entire software stack.
+    EL 8 included a major version upgrade of Python from 2 to 3, resulting in major development efforts by the team and
+    coordination of external software providers.
 
 -   **Streamlining the release process:** to get new software into the hands of site administrators more quickly, the
```

Comment:
I wonder if this bullet is best combined with the container one. Not all DevOps is containers, right?